apps/starccm/readme.md

```diff
@@ -71,8 +76,8 @@ Environment variables can be passed to the PBS job with the `-v` flag.
 
 Submit a job as follows (remembering to substitute your PoD key value):
 
-    qsub -l select=2:ncpus=60:mpiprocs=60:place=scatter:excl \
-        -v PODKEY=#INSERT_POD_KEY# \
+    qsub -l select=2:ncpus=60:mpiprocs=60,place=scatter:excl \
+        -v PODKEY=#INSERT_POD_KEY# -v CASE=#Path to civil.sim# \
```

Comment:
CASE is just for the name of the file excluding the path and extension.  DATA_DIR was for where the sim files are located.

Reply:
from readme we are copying the sim file to headnode here. we need to fix that too as the script expects it in DATA_DIR as Paul mentions.
_azhpc-scp civil.sim hpcuser@headnode:._